examples/static-site-gen/platform/src/lib.rs

```diff
@@ -210,21 +216,34 @@ fn process_file(input_dir: &Path, output_dir: &Path, input_file: &Path) -> Resul
     // And track a little bit of state
     let mut in_code_block = false;
     let mut is_roc_code = false;
-
+    let ps : syntect::parsing::SyntaxSet = SyntaxSet::load_defaults_newlines();
+    let ts : syntect::highlighting::ThemeSet = ThemeSet::load_defaults();
+    
     for event in parser {
         match event {
             pulldown_cmark::Event::Code(cow_str) => {
```

Comment:
Perhaps `code_str` might be a better alternative to `cow_str`?

Reply:
Changed